build.sbt

```diff
@@ -108,6 +109,7 @@ lazy val core = project
       "io.laserdisc"                     %% "log-effect-zio"                 % V.`log-effect`,
       "is.cir"                           %% "ciris"                          % V.ciris,
       "org.apache.kafka"                  % "kafka-clients"                  % V.kafka,
+      "org.scalatestplus"                %% "scalatestplus-mockito"          % V.mockito       % Test,
```

Comment:
can we get rid of this at next iteration? I believe it's only needed to mock out the recent addition of the AdminClient in the core

Reply:
Yeah, I surely hate it but there are several direction we could go towards. Do we want also a fake admin so we can test the automatic rewind? I tried and it wasn't trivial so I dropped it, but it's certainly possible. Do we want to just remove the dependency? In this case we need a homebrew mock, which won't be easy given that, unlike producer and consumer, this doesn't have an interface.

Reply:
I'd go with the former, but aside for not being trivial I guess we also need to abstract away the construction of the `AdminClient` ourselves in `tamer.Tamer`, so we can replace this https://github.com/zio/zio-kafka/blob/007bebb58d8542e4c49bd6078b3e9bdea559d12e/src/main/scala/zio/kafka/admin/AdminClient.scala#L625-L630 with our FakeAdminClient, right?

Let's think about it a bit more and maybe create a card for it?

Reply:
Ok let's also talk about what kind of abstraction we want. I would also evaluate directly contributing by enveloping the AdminClient into service so that everything become uniform and homogeneous.

Reply:
Agreed. We like for sure the idea of having an `Admin` service with a live implementation backed by the current java AdminClient, so let's try and contribute to that!